.github/CONTRIBUTING.md

```diff
@@ -20,33 +20,7 @@ If you meet the above criteria, you can submit issues with our [GitHub issue tra
 
 ## Feature Requests
 
-You can also use the issue tracker to request a new feature or enhancement. Even if you want to implement the feature yourself, please first submit an issue detailing your proposal so that we may discuss it with you and the community before moving forward. Please use [labels](#usage-of-labels) when creating feature requests.
-
```

Comment:
We can put this back, but seeing as we don't really use the labels it's not very accurate besides `Contributions Welcome` which is talked about in a different section.